modules/setup/setup.js

```diff
@@ -16,87 +17,74 @@ class Setup {
               }
         });
     }
 
     startNodes() {
         Logger.verbose('Command : tezster start-nodes');
         child_process.exec(`docker images ${IMAGE_TAG} --format {{.Repository}}:{{.Tag}}:{{.Size}}`,
-        (error, _stdout, _stderr) => {
-            if (_stdout === `${IMAGE_TAG}:2.96GB\n`) {
+        (error, stdout, stderr) => {
+            if (stdout === `${IMAGE_TAG}:2.96GB\n`) {
 
                 child_process.exec(`docker ps -a -q  --filter ancestor=${IMAGE_TAG} --format {{.Image}}:{{.Names}}`,
-                (error, _stdout, _stderr) => {
-                    let self = this;
-                    if (_stdout.includes(`${IMAGE_TAG}:${CONTAINER_NAME}\n`)) {
-                        const container = docker.getContainer(CONTAINER_NAME);
-                        container.start(function (error, data){
-                            if(error) {
-                                Logger.verbose(`Error while starting nodes: ${error}`);
-                                Logger.info('Nodes are already running....');
-                                Logger.warn(`If still facing issue run, 'tezster get-logs' or try restarting the nodes after running 'tezster stop-nodes'....`);
-                            } else {
-                                self.startNodesProgressBar();
-                            }
-                        });
+                (error, stdout, stderr) => {
+
+                    if (stdout.includes(`${IMAGE_TAG}:${CONTAINER_NAME}\n`)) {
+                        this.startExistingContainer();
                     } else if (error) {
-                        Logger.verbose(`Error while starting nodes error: ${error}`);
+                        Helper.errorLogHandler(`Error occurred while starting the nodes ${error}`, `Error occurred while starting the nodes, try restarting the nodes after running 'tezster stop-nodes'`);
                     } else {                         
-                          this.runContainer();
+                        this.runContainer();
                     }
                 });
-            } else {
-                  Logger.verbose(`Error while starting nodes error: ${error}`);
-                  Logger.verbose(`Error while starting nodes std_err: ${_stderr}`);   
-                  Logger.error(`No inbuilt nodes found on system. Run 'tezster setup' comamnd for build the nodes.`);
-              }
+
+            } else if(error = error || stderr){
+                Helper.errorLogHandler(`Error occurred while starting the nodes: ${error}`, `No inbuilt nodes found on system. Run 'tezster setup' command for build the nodes.`);
+            }
         });
     }
 
     stopNodes() {
         Logger.verbose('Command : tezster stop-nodes');
-        child_process.exec(`docker ps -a -q --format {{.Image}}`,(error, _stdout, _stderr) => {
-            if (_stdout.includes(`${IMAGE_TAG}\n`)) {
+        child_process.exec(`docker ps -a -q --format {{.Names}}`,(error, stdout, stderr) => {
```

Comment:
Why a variable is named as 'NAMES' ?

Reply:
We didn't named this, it is docker nomenclature. We can see container name column named as 'NAMES'.